src/HW4/Student.java

```diff
@@ -0,0 +1,10 @@
+package HW4;
+
+public class Student {
```

Comment:
It's better idea to create constructor:
 public Student(String lastName, double mathMark, double economicMark,
                 double foreignLangMark) {
    this.lastName = lastName;
    this.mathAvgMark = mathMark;
    this.economicAvgMark = economicMark;
    this.foreignLangAvgMark = foreignLangMark;
  }